documentation/src/settings/ffsend_url.md

```diff
@@ -0,0 +1,3 @@
+### Firefox Send URL
```

Comment:
As Send is not affiliated with Mozilla and/or Firefox anymore, perhaps we should just call it Send? (And also change any references in the code). Or maybe call it something like:

Send (previously: Mozilla Send)

or something like that